index.html

```diff
@@ -47,160 +57,211 @@ <h1 class="introduction-title">Hey there, I’m Matama <span class="title-green"
+          <li><a href="#" class="anc card-a3">Javscript</a></li>
+          <li><a href="#" class="anc card-a4">html</a></li>
+        </ul>
+        <a href="#" class="tag">
+          <button type="button" class="btn btn-success see-button">See Project</button>
+        </a>
+      </div>
 
-    <div class="card">
-      <div class="card-image car"></div>
-      <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
-      <ul class="alist">
-        <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
-        <li><a href="#" class="anc card-a2">CSS</a></li>
-        <li><a href="#" class="anc card-a3">Javscript</a></li>
-        <li><a href="#" class="anc card-a4">html</a></li>
-      </ul>
-      <a href="#" class="tag">
-        <button type="button" class="btn btn-success see-button">See Project</button>
-      </a>
-    </div>
+      <div class="card">
+        <div class="card-image car"></div>
+        <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
+        <ul class="alist">
+          <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
+          <li><a href="#" class="anc card-a2">CSS</a></li>
+          <li><a href="#" class="anc card-a3">Javscript</a></li>
+          <li><a href="#" class="anc card-a4">html</a></li>
+        </ul>
+        <a href="#" class="tag">
+          <button type="button" class="btn btn-success see-button">See Project</button>
+        </a>
+      </div>
 
- <!-- About me -->
-    <div class="about-me">
-      <h1 class="intro">About Me</h1>
-      <p class="about-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
-don’t hestiate to contact me.</p>
-      <img src="images/about.svg" alt="">
-      <div class="text-center">
+      <div class="card">
+        <div class="card-image car"></div>
+        <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
+        <ul class="alist">
+          <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
+          <li><a href="#" class="anc card-a2">CSS</a></li>
+          <li><a href="#" class="anc card-a3">Javscript</a></li>
+          <li><a href="#" class="anc card-a4">html</a></li>
+        </ul>
+        <a href="#" class="tag">
+          <button type="button" class="btn btn-success see-button">See Project</button>
+        </a>
+      </div>
+      </div>
+
+      <!-- About me -->
+      <div class="about-me">
+        <div class="about-me-text">
+        <h1 class="intro">About Me</h1>
+        <p class="about-text"> I can help you build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need coded,
+          don’t hestiate to contact me.</p>
+        </div>
+        <div class="text-center">
           <button type="button" class="btn btn-success resume-button">Get My Resume</button>
+        </div>
```

Comment:
- [ ] I would kindly suggest you follow the Figma template design as much as possible, As currently the `About section` of your project is missing one of the `background image` as we have it in the Figma template and the second one is overflowing with the `text`, Kindly check the Figma template to rectify it.

Your Design 👇 

![dan abt](https://user-images.githubusercontent.com/78634841/206759055-b2da4713-ce2d-4ca8-9409-6fafab08e31f.png)

Figma Design 👇 

![abt real](https://user-images.githubusercontent.com/78634841/206759119-07384124-72d8-4a5a-b523-159d44d69a80.png)